pkg/lib/journal.js

```diff
@@ -230,7 +234,34 @@
                 };
             if (count > 1)
                 parts['count'] = count;
-            if (prio < 4)
+            if (ident === 'abrt-notification') {
+                // Problem can be identified by 3 elements. Most likely the first, but in some
+                // cases other two can be used as well.
+                var all_p = [entry['PROBLEM_DIR'], entry['PROBLEM_DUPHASH'], entry['PROBLEM_UUID']];
+                var p;
+                var known = false;
+                for (var i = 0; i < all_p.length; i++) {
+                    if (all_p[i] in displayable_problems) {
+                        p = all_p[i];
+                        known = true;
+                        break;
+                    }
+                }
+                //Do not render unknown (deleted) problems
+                if (! known)
+                    return "";
```

Comment:
> Yes. The journal is a time-line of events....

Yeah, this makes a better sense.

> The list of journal entries is not the place to show this kind of summary...

Ok than. The design specified that this should dispaly the problem count and I liked that idea. However, if you believe it is not a good feature, we can omit it easily.

> ...  "Notifications" UI ...

I like this! When it gets implemented one day, it would be useful to inform about ABRT problems there.